src/status_im/contexts/wallet/collectible/utils.cljs

```diff
@@ -7,3 +10,37 @@
       first
       :balance
       js/parseInt))
+
+(defn- get-opensea-network-name
+  [chain-id test-networks-enabled? is-goerli-enabled?]
+  (let [network-kw   (network-utils/id->network chain-id)
+        network-name (name network-kw)
+        mainnet?     (= :mainnet network-kw)]
+    (cond (and test-networks-enabled? is-goerli-enabled?)
+          (if mainnet?
+            constants/goerli
+            (str network-name "-" constants/goerli))
+          test-networks-enabled?
+          (if mainnet?
+            constants/sepolia
+            (str network-name "-" constants/sepolia))
+          :else
+          (if mainnet?
+            constants/ethereum
+            network-name))))
+
+(defn- get-opensea-base-url
+  [test-networks-enabled?]
+  (cond
+    test-networks-enabled? config/opensea-tesnet-link
+    :else                  config/opensea-link))
```

Comment:
Can this be an `if`? or are we expecting to consider more cases in a near future?